tests/models/test_poromechanics.py

```diff
@@ -455,6 +455,75 @@ def test_pull_south_positive_reference_pressure():
     assert np.allclose(p_vals, p_vals_ref + 1)
 
 
+def test_pull_south_positive_reference_pressure_with_multiphysics_norms():
```

Comment:
Isn't this rather a test of the solution strategy?

Reply:
It's not entirely clear to me what this test is supposed to verify. Please elaborate.

Reply:
Test is removed.